src/server/mod.rs

```diff
@@ -109,12 +109,12 @@ impl Server {
             raw_filter.update().await?
         }
         let filter = Arc::new(RwLock::new(raw_filter));
-        spawn_updater(&filter);
-        let tiles_cache = cache::TilesCache::new(TILES_CACHE_INITIAL_CAPACITY);
         let req = reqwest::Client::builder()
             .connect_timeout(Duration::from_secs(settings.connect_timeout))
```

Comment:
let's add a `.timeout` call here, ensuring it's already set as a default everywhere we reuse this client. I think we'd 'overwrite' this timeout value in a couple places (like in `get_tiles`) but it doesn't really matter.

Reply:
Ah, well, if we do that, we should specify both `connection_timeout` and `timeout` since they are used independently. I'll add the code to do that.

Reply:
Agreed, looks like both `timeout` and `connection_timeout` are needed for all the Reqwest uses in Contile. The same applies to Merino once it needs to talk to external services at scale.